pkg/ovssubnet/controller/kube/bin/openshift-sdn-kube-subnet-setup.sh

```diff
@@ -147,11 +141,15 @@ function setup() {
     echo "export OPENSHIFT_CLUSTER_SUBNET=${cluster_network_cidr}" >> "/etc/openshift-sdn/config.env"
 
     # delete unnecessary routes
-    delete_local_subnet_route lbr0
+    delete_local_subnet_route lbr0 || true
```

Comment:
We should *not* tolerate deletion of lbr0 route.
If we allow creating the lbr0 route then we will run into this issue: https://bugzilla.redhat.com/show_bug.cgi?id=1275388
I thought 2 secs timeout was sufficient, I will do further investigation.